chrome/popup/popup.css

```diff
@@ -208,7 +208,8 @@ panel[type="arrow"][side="right"]
 	margin-bottom: -44px;
 }
 
-panel:not(.tooltip-xul-wrapper) .panel-arrowcontent
+panel:not(.tooltip-xul-wrapper) .panel-arrowcontent,
```

Comment:
I left this line in here but if the single `.panel-arrowcontent` selector doesn't break anything we can remove this line.